docs/open-api.yaml

```diff
@@ -128,6 +160,27 @@ components:
         sameAs:
           type: string
           nullable: true
+    CreateJournal:
+      type: object
+      properties:
+        name:
+          type: string
+          description: Name of the journal
+        pIssn:
+          type: string
+          nullable: true
+          description: The publication ISSN
+          pattern: ^[0-9]{4}-[0-9]{3}[0-9X]$
+        eIssn:
+          type: string
+          nullable: true
+          description: The electronic ISSN
+          pattern: ^[0-9]{4}-[0-9]{3}[0-9X]$
+        url:
+          type: string
+          format: uri
+          nullable: true
+          description: The url to the journal
```

Comment:
I doubt that these map to the relevant ontology

Reply:
I've updated pIssn to printIssn and eIssn to onlineIssn. 
Do I keep the url, or is it 'sameAs'?

Reply:
Let us create a separate task for updating the ontology. See my other comment regarding `url`.